coderadar-server/coderadar-core/src/test/java/org/wickedsource/coderadar/metricquery/rest/tree/MetricsTreeControllerTest.java

```diff
@@ -61,11 +61,11 @@ public void queryMetricsTree() throws Exception {
                             .type(JsonFieldType.STRING)
                             .description(
                                 "Either 'MODULE' if this node describes a module which can have child nodes or 'FILE' if this node describes a file (which has no child nodes)."),
-                        fieldWithPath("metrics")
+                        subsectionWithPath("metrics")
```

Comment:
why did you change `fieldWithPath` to `subsectionWithPath`?

Reply:
Otherwise I get the following error:

```
org.springframework.restdocs.snippet.SnippetException: The following parts of the payload were not documented:
{
  "metrics" : {
    "metric1" : 38,
    "metric2" : 28,
    "metric3" : 19
  }
}
```

I suspect that is because we don't document `metric1`, `metric2` and `metric3` in `requestFields`.
Seeing as how the test passes all other checks (including cheking the values of the metrics), I thought this change was appropriate.

Reply:
OK